vendor/mooltipass/mcCombinations.js

```diff
@@ -359,6 +359,41 @@ var extendedCombinations = {
             }
         }
     },
+    protonmail: function (forms) {
```

Comment:
could you rename it to protonmailuservoice, as it isn't for the main protonmail page?

Reply:
Of course! Will do!